examples/go/main.go

```diff
@@ -14,8 +13,8 @@ func main() {
 		if err != nil {
 			return err
 		}
-
-		registry, err := registrygeoreplication.NewRegistryGeoReplication(ctx, "registry", &registrygeoreplication.RegistryGeoReplicationArgs{
+		azure-quickstart-acr-geo-replication
+		registry, err := acr.NewRegistry(ctx, "registry", &azure.RegistryGeoReplicationArgs{
```

Comment:
The path for args and creation needs to agree. 
```suggestion
		registry, err := acr.NewRepository(ctx, "registry", &acr.RegistryGeoReplicationArgs{
```